thijsvanloef/palworld-server-docker
# Discord: Add support for Mentions

## Context 
Fixes https://github.com/thijsvanloef/palworld-server-docker/issues/300

Changing the way the Discord messages are built, Allowing you to add a user ID to the discord message to mention people or roles.

## Choices

Move the message from the Title to the description and add the script name as a title.

## Test instructions

1. Build the container
2. Set DISCORD_WEBHOOK_URL and change any of the MESSAGES (e.g. DISCORD_PRE_START_MESSAGE) to include a mention using `<@user_id>`
3. Check if you are mentioned in the Discord Channel

## Checklist before requesting a review
